docs/language/learn-ql/cpp/introduce-libraries-cpp.rst

```diff
@@ -1,20 +1,25 @@
-Introducing the CodeQL libraries for C/C++
-==========================================
+CodeQL libraries for C and C++
+==============================
 
-Overview
---------
+Explore the standard CodeQL libraries for C and C++.
 
-There is an extensive library for analyzing CodeQL databases extracted from C/C++ projects. The classes in this library present the data from a database in an object-oriented form and provide abstractions and predicates to help you with common analysis tasks.  The library is implemented as a set of QL modules, that is, files with the extension ``.qll``. The module ``cpp.qll`` imports all the core C/C++ library modules, so you can include the complete library by beginning your query with:
+About the CodeQL libraries for C and C++
+----------------------------------------
+
+There is an extensive library for analyzing CodeQL databases extracted from C/C++ projects. The classes in this library present the data from a database in an object-oriented form and provide abstractions and predicates to help you with common analysis tasks. 
+The library is implemented as a set of QL modules, that is, files with the extension ``.qll``. The module ``cpp.qll`` imports all the core C/C++ library modules, so you can include the complete library by beginning your query with:
 
 .. code-block:: ql
 
    import cpp
 
 The rest of this topic summarizes the available CodeQL classes and corresponding C/C++ constructs.
 
-NOTE: You can find related classes and features using the query console's auto-complete feature.  You can also press *F3* to jump to the definition of any element; library files are opened in new tabs in the console.
+.. pull-quote:: Note
+
+   You can find related classes and features using the query console's auto-complete feature.  You can also press *F3* to jump to the definition of any element. Library files are opened in new tabs in the console.
```

Comment:
Not required for this PR, but this note should probably have been deleted or clarified when we moved the documentation out of LGTM. We should probably check other languages for similar notes.